lib/fuse_lowlevel.c

```diff
@@ -2863,7 +2876,8 @@ void fuse_lowlevel_help(void)
 	printf(
 "    -o allow_other         allow access by all users\n"
 "    -o allow_root          allow access by root\n"
-"    -o auto_unmount        auto unmount on process termination\n");
+"    -o auto_unmount        auto unmount on process termination\n"
+"    -o no_interrupt        do not process interrupt request");
 }
```

Comment:
I think that would be rather confusing for end users what this actually means. My suggestion is to add a function 
`fuse_session_disable_interrupts(struct fuse_session *se)` and then to set this in the file system after creating the session. What do you think?

Reply:
And I definitely like the idea.

Reply:
Thanks for your suggestions!
I think it's more reasonable to use the function `fuse_session_disable_interrupts(struct fuse_session *se)` since this `no_interrupt` option is related to the fuse file system implementation, not an option for end users. 
The code is updated according to your suggestions.